metrics.go

```diff
@@ -11,32 +11,42 @@ import (
 // with values relative to that last second.
 type MetricsBag map[string]interface{}
 
-// AddRawCounter adds a counter that always increments.
+// AddRawCounter adds a metric of type counter to the bag and sets its current value.
+// A counter is a cumulative metric that can only increase.
+// To be meaningful, `value` must be positive.
```

Comment:
In general we don't use backticks in comments as anyway they're not rendered in GoDoc.
So both of the following suggestions are ok (I personally prefer the first one):
```suggestion
// To be meaningful, value must be positive.
```
```suggestion
// To be meaningful, 'value' must be positive.
```